tests/build/test_launch_script.py

```diff
@@ -61,7 +61,6 @@
         "prompt_tuning_init": "RANDOM",
         "num_virtual_tokens": 8,
         "prompt_tuning_init_text": "hello",
-        "tokenizer_name_or_path": MODEL_NAME,
```

Comment:
This will trigger custom tokenizer and does not add pad tokens which would fail this test, so had to remove it.